### DISCUSSION TITLE
Do you plan on supporting multi-language for medical model like the general model in speech to text

### DISCUSSION THREAD

**Discussion by 3mawy** [deepgram/community#1012] [General help] [answered]
We would like to have multi-language support in Nova-2 Medical, similar to what is available in Nova-2 or Nova-2 General. Are there any plans to implement this feature in the near future?

**Comment by deepgram-community** [comment]
Hey there! It looks like you haven't connected your GitHub account to your Deepgram account. You can do this at https://community.deepgram.com - being verified through this process will allow our team to help you in a much more streamlined fashion.

**Comment by deepgram-community** [comment]
It looks like we're missing some important information to help debug your issue. Would you mind providing us with the following details in a reply?

- The programming language you are working in (e.g. JavaScript, Python).
- The deepgram product you are using (e.g Speech to Text, Agent API)
- A request ID that triggered your error or issue.

**Comment by deepgram-community** [comment, marked as answer]
At this time I can't give specifics on when our medical models would support more languages,  but we are always looking to add new language support to our models.

> ![avatar](https://avatars.slack-edge.com/2023-02-20/4855338233600_fad9174023ee3f640ce1_32.png) This message was sent by John Vajda from Deepgram, via our community automation.
